recipes.js

```diff
@@ -24,18 +23,65 @@ export const createNodeRecipes = () => {
     }).appendTo($('.recipes__list'))
 }
 
+const sendRecipeToBackEnd = recipeName => {
+    const sendRecipe = fetch('http://localhost:3001/api/recipes', {
+        method: 'POST',
+        body: JSON.stringify({
+            recipeName: recipeName,
+            ingredients: ingredientsAddedToRecipeArray
+        }),
+        headers: {
+            "Content-Type": "application/json"
+        }
+    })
+        .then(res => {
+            if (res.ok) {
+                return res.json()
+            }
+
+            throw error
+        })
+        .catch(error => {
+            alert(error)
```

Comment:
you should also return eg. null, so your recipe returned from function will be null, you can handle it in the next function